### ISSUE TITLE
Make POD dashboard & /metrics consistent or explain why not on both

### ISSUE THREAD

**Issue by rebeccawilliams** [GSA/datagov-wptheme#647]
See discrepancies between: 
- http://labs.data.gov/dashboard/offices
- http://www.data.gov/metric/federalagency/dataset-published-per-month and 

e.g. at the date of filing this ssue (with Metrics last being updated on 6/16; and POD's 5/31 milestone)

> DOI's totals: POD = 84,379, /Metrics = 28,608
> Commerce's totals: POD = 487, /Metrics = 38,705

Reasons for discrepancies =
- geospatial harvests 
- dataset URL v distribution counts (?)
- update frequencies 
- for all of Data.gov: local and non CFO Act harvests
- ?

**Comment by rebeccawilliams** [comment]
^ This was brought to my attention by @tscarolin.

**Comment by philipashlock** [comment]
yeah, they measure different things so we should clarify. Other than geospatial harvest sources that are not yet represented in agency data.json files another potential discrepancy is collections, ticketed with https://github.com/GSA/project-open-data-dashboard/issues/98

**Comment by rebeccawilliams** [comment]
Can this be noted on data.gov/metrics as well?

**Comment by theresaanna** [comment]
Questions:
- Where are all the places this difference should be noted?
- What language should be used?
